server/models/MealItem.py

```diff
@@ -0,0 +1,39 @@
+from . import db, bcrypt
+from marshmallow import Schema
+
+
+class MealItem(db.Model):
+    __tablename__ = 'meal_items'
+
+    id = db.Column(db.Integer, primary_key=True)
+    userId = db.Column(
+        db.Integer,
+        db.ForeignKey('users.id', ondelete='CASCADE'),
+        nullable=False,
+    )
+    name = db.Column(db.String(100), nullable=False)
+    description = db.Column(db.Text, nullable=True)
+    # to access chef info from a meal item
+    user = db.relationship("User")
+
+    def __init__(self, userId, name, description=""):
+        self.userId = userId
+        self.name = name
+        self.description = description
+
+    def __repr__(self):
+        return f"<Meal #{self.id}: {self.name}>"
+
+    def save_to_database(self):
```

Comment:
MealItem and User models have the save save_to_database method. Is there a way to write this once and somehow have it used by both models.

Reply:
You mean like helper function? Or, a separate class that they both inherit from?